### PR TITLE
feat: revert gas token naming

### PR DESCRIPTION
**Pull request by 0xShaito**

### CODE REVIEW

**Review by excaliborr — DISMISSED** [review]

**Review by excaliborr — DISMISSED** [review]
lgtm

**Review by 0xGorilla — DISMISSED** [review]

File: solidity/contracts/XERC20Factory.sol [hunk 1]
```diff
@@ -53,22 +53,22 @@ contract XERC20Factory is IXERC20Factory {
    * @dev When deploying a lockbox for the gas token of the chain, then, the base token needs to be address(0)
    * @param _xerc20 The address of the xerc20 that you want to deploy a lockbox for
    * @param _baseToken The address of the base token that you want to lock
-   * @param _isGasToken Whether or not the base token is the native (gas) token of the chain. Eg: MATIC for polygon chain
+   * @param _isNative Whether or not the base token is the native (gas) token of the chain. Eg: MATIC for polygon chain
```
Comment (0xGorilla) [inline solidity/contracts/XERC20Factory.sol:57]: where is the `@return`?

File: solidity/contracts/XERC20Factory.sol [hunk 2]
```diff
@@ -112,11 +112,11 @@ contract XERC20Factory is IXERC20Factory {
   function _deployLockbox(
```
Comment (0xGorilla) [inline solidity/contracts/XERC20Factory.sol:124]: where is the natspec?

Comment (0xGorilla) [reply solidity/contracts/XERC20Factory.sol:124]: and the `@return` on _deployXERC20

**Review by excaliborr — DISMISSED** [review]

**Comment by just-a-node** [comment]
Question - I checked out this branch to deploy a testnet xERC20 and noticed that `XERC20Deploy.sol` has a hardcoded factory address. Are we not using the `FactoryAddress.txt` anymore?

**Comment by 0xShaito** [comment]
@just-a-node No. We felt it was unnecessary complexity as 99% of the time protocols shouldn't change the factory address and only when a new factory is deployed it should be updated (probably by us or a fork) so we removed it.

**Comment by just-a-node** [comment]
> @just-a-node No. We felt it was unnecessary complexity as 99% of the time protocols shouldn't change the factory address and only when a new factory is deployed it should be updated (probably by us or a fork) so we removed it.

Okay, that makes sense. Could we remove it then and also update the factory? The current one in `XERC20Deploy.sol` is a factory with the `IS_GAS_TOKEN` naming.

**Comment by 0xShaito** [comment]
Good call @just-a-node . Updated! Thank you

**Review by excaliborr — APPROVED** [review]
lgtm!

**Review by just-a-node — APPROVED** [review]

**Comment by just-a-node** [comment]
Optimistically merging now so forks will use the original interface.

**Pull request merged**